test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/AssemblyEnumeratorTests.cs

```diff
@@ -102,7 +102,7 @@ public void GetTypesShouldReturnSetOfDefinedTypes()
         mockAssembly.Setup(a => a.DefinedTypes).Returns(expectedTypes);
 
         var types = AssemblyEnumerator.GetTypes(mockAssembly.Object, string.Empty, _warnings);
-        VerifyCollectionsAreEqual(expectedTypes.ToArray(), types);
+        Verify(expectedTypes.ToArray().SequenceEqual(types));
```

Comment:
Nit: `SequenceEqual` working with `IEnumerable<T>` so virtually all collections.

```suggestion
        Verify(expectedTypes.SequenceEqual(types));
```